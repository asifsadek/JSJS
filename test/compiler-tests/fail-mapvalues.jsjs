val m = {"foo": 10, "bar": 20, "baz": 30};
val xs: list string = Map.values(m); // fail: return type is list num
