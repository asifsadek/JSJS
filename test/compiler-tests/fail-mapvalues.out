Type error: expected value of type 'list string', got a value of type 'list num' instead
